shared/teams/team/container.js

```diff
@@ -49,6 +50,17 @@ const mapStateToProps = (state: TypedState, {routeProps, routeState}): StateProp
     ['teams', 'teamNameToImplicitAdminUsernames', teamname],
     I.Set()
   )
+  // We had to request every subteam of the top-level team, rather than just
+  // child subteams of the subteam we care about.  Here's where we fix that up.
+  const subteams = state.entities
+    .getIn(['teams', 'teamNameToSubteams', teamname], I.Set())
+    .filter(team => team.startsWith(teamname + '.'))
+  const subTeamsProps = subteams.map(subteam => ({
+    key: subteam,
```

Comment:
just pass teamname as the key in the list and you can get rid of this property

Reply:
done